sql/importer: TestImportDefaultWithResume failed

sql/importer.TestImportDefaultWithResume [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7612050?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7612050?buildTab=artifacts#/) on master @ [cfb5ae9a96e1770daa4aef1615a46e212b561a84](https://github.com/cockroachdb/cockroach/commits/cfb5ae9a96e1770daa4aef1615a46e212b561a84):


```
=== RUN   TestImportDefaultWithResume
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/73dfce30b9a5630b1b4dabed3c94b32c/logTestImportDefaultWithResume1843151697
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestImportDefaultWithResume
    import_stmt_test.go:4711: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/73dfce30b9a5630b1b4dabed3c94b32c/logTestImportDefaultWithResume1843151697
--- FAIL: TestImportDefaultWithResume (0.91s)
=== RUN   TestImportDefaultWithResume/nextval
    import_stmt_test.go:4698: not enough memory available to create a BulkAdder: root: memory budget exceeded: 33554432 bytes requested, 101892096 currently allocated, 134217728 bytes in budget
    --- FAIL: TestImportDefaultWithResume/nextval (0.43s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportDefaultWithResume.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21660

sql/importer.TestImportDefaultWithResume [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7623080?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7623080?buildTab=artifacts#/) on master @ [f0554bc215e31ff53e644936ba645bd895ec0d15](https://github.com/cockroachdb/cockroach/commits/f0554bc215e31ff53e644936ba645bd895ec0d15):


```
=== RUN   TestImportDefaultWithResume
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/73dfce30b9a5630b1b4dabed3c94b32c/logTestImportDefaultWithResume3895174889
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestImportDefaultWithResume
    import_stmt_test.go:4711: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/73dfce30b9a5630b1b4dabed3c94b32c/logTestImportDefaultWithResume3895174889
--- FAIL: TestImportDefaultWithResume (0.73s)
=== RUN   TestImportDefaultWithResume/nextval
    import_stmt_test.go:4698: not enough memory available to create a BulkAdder: root: memory budget exceeded: 33554432 bytes requested, 101943296 currently allocated, 134217728 bytes in budget
    --- FAIL: TestImportDefaultWithResume/nextval (0.37s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportDefaultWithResume.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/importer/import_stmt_test.go
+++ b/pkg/sql/importer/import_stmt_test.go
@@ -4582,6 +4582,7 @@ func TestImportDefaultWithResume(t *testing.T) {
 					BulkAdderFlushesEveryBatch: true,
 				},
 			},
+			SQLMemoryPoolSize: 1 << 30, // 1 GiB
 		})
 	registry := s.JobRegistry().(*jobs.Registry)
 	ctx := context.Background()

